engine/client/voice.c

```diff
@@ -440,7 +439,7 @@ Feed the decoded data to engine sound subsystem
 static void Voice_StartChannel( uint samples, byte *data, int entnum )
 {
 	SND_ForceInitMouth( entnum );
-	S_RawEntSamples( entnum, samples, voice.samplerate, voice.width, VOICE_PCM_CHANNELS, data, 255 );
+	S_RawEntSamples( entnum, samples, voice.samplerate, voice.width, VOICE_PCM_CHANNELS, data, bound( 0, 255 * voice_scale.value, 255 ));
```

Comment:
Interesting how we can just increase the volume past 255 for rawchannels and it just works?

Because normal channel volume is capped at 255.

Reply:
voice_scale seems to be value in range `[0.0; 1.0]`

Reply:
Then let's cap it in that range then?

Reply:
This already works in similar way, isn't it?

Reply:
Ah, you're right.